app/models/employment/Benefits.scala

```diff
@@ -73,14 +73,13 @@ case class Benefits(accommodation: Option[BigDecimal] = None,
   val assetsDetailsPopulated: Boolean =
     assets.isDefined || assetTransfer.isDefined
 
-  val hasBenefitsPopulated: Boolean = {
+  val hasBenefitsPopulated: Boolean =
```

Comment:
It might be nice to have curly braces here for visual clarity. I know they aren't necessary though

Reply:
I think it is clear enough and removes some noise in the code. Further this is a value, not a method so I think we should not have a block.

Reply:
Yeah looking in the code in intelliJ it is clear enough. I'm happy to leave it as is.